tests/pytype_test.py

```diff
@@ -149,6 +153,23 @@ def find_stubs_in_paths(paths: Sequence[str]) -> list[str]:
     return filenames
 
 
+def _is_supported_stdlib_version(module_versions: SupportedVersionsDict, filename: str) -> bool:
+    parts = _get_relative(filename).split(os.path.sep)
+    if parts[0] != "stdlib":
+        return True
+    module_name = _get_module_name(filename)
+    min_version, max_version = _supported_versions_for_module(module_versions, module_name)
+    return min_version <= sys.version_info <= max_version
+
+
+def _supported_versions_for_module(module_versions: SupportedVersionsDict, module_name: str) -> tuple[VersionTuple, VersionTuple]:
+    while "." in module_name:
+        if module_name in module_versions:
+            return module_versions[module_name]
+        module_name = ".".join(module_name.split(".")[:-1])
+    return module_versions[module_name]
```

Comment:
Could you instead move `supported_versions_for_module` from `mypy_test.py` to `_utils.py` and use the common function?

Reply:
Good idea. Moved and imported from pytype and mypy tests.